integration_tests/util.go

```diff
@@ -52,6 +52,10 @@ func LndHubTestServiceInit(lndClientMock lnd.LightningClientWrapper) (svc *servi
 		JWTRefreshTokenExpiry: 3600,
 		LNDAddress:            mockLNDAddress,
 		LNDMacaroonHex:        mockLNDMacaroonHex,
+		MaxReceiveAmount:      1000000,
+		MaxSendAmount:         100000,
+		LnurlAPIPrefix:        "testnet",
+		LnurlDomain:           "example.com",
```

Comment:
I think it's better when `LnurlAPIPrefix` and `LnurlDomain` are combined into a single variable (`LnurlDomain`).

All occurences in the code use `LnurlAPIPrefix` and `LnurlDomain` together (as `LnurlAPIPrefix + "." + LnurlDomain`) except this one:

``` golang
responseBody.Metadata = lnurlDescriptionHash(user.Nickname, controller.svc.Config.LnurlDomain)
```

and even there I think the full domain name should be used (including prefix).